libsolidity/codegen/CompilerUtils.cpp

```diff
@@ -160,7 +160,15 @@ void CompilerUtils::encodeToMemory(
 	TypePointers targetTypes = _targetTypes.empty() ? _givenTypes : _targetTypes;
 	solAssert(targetTypes.size() == _givenTypes.size(), "");
 	for (TypePointer& t: targetTypes)
+	{
+		solAssert(
+			t->mobileType() &&
+			t->mobileType()->interfaceType(_encodeAsLibraryTypes) &&
+			t->mobileType()->interfaceType(_encodeAsLibraryTypes)->encodingType(),
+			"Encoding type " + t->toString() + " not yet implemented."
```

Comment:
I suggest something like `"Encoding type [" + t->toString() + "] not yet implemented."` for better readability.